search-helpers/geosearch-v2.js

```diff
@@ -0,0 +1,29 @@
+const rp = require('request-promise');
+
+const geosearchV2 = (string, req) => {
+  const referer = req.header('Referer');
+  const geoSearchAPICall =
+   `https://geosearch.planninglabs.nyc/v2/autocomplete?boundary.rect.min_lon=-74.292297&boundary.rect.max_lon=-73.618011&boundary.rect.min_lat=40.477248&boundary.rect.max_lat=40.958123&text=${string}`;
+
+  return rp(geoSearchAPICall, {
+    headers: {
+      'Referer': `labs-search-api--${referer}`, // eslint-disable-line
+    },
+  })
+    .then(res => JSON.parse(res))
+    .then(json => json.features.filter(feature => feature.properties.borough))
+    .then(json => json.map((feature) => {
+      const { label, pad_bbl: bbl } = feature.properties;
```

Comment:
Does this actually work? The response from v2 endpoint should have the bbl at `feature.properties.addendum.pad.bbl`